datasets/movielens/data.json

```diff
@@ -0,0 +1,244 @@
+{
+  "@context": {
+    "@vocab" : "https://schema.org/",
```

Comment:
@pierrot0 changed the titanic example from using `@vocab` to explicitly using `"sc": "https://schema.org/",`
in https://github.com/mlcommons/datasets_format/commit/35472310fd323549ad3ad574807ed09c20efd509.
Both methods are fine of course, but I like the explicit approach

Reply:
Done. I think we still need the @vocab declaration so that we don't have to prefix all schema.org names (e.g, name, url, description, etc.)